types/tizen-common-web/humanactivity.d.ts

```diff
@@ -0,0 +1,170 @@
+import { ErrorCallback } from './tizen';
+
+export interface HumanActivityMonitorManager {
+    // tslint:disable-next-line no-unnecessary-generics as parameter types are contravariant, and use of generics allows to infer type for callback parameter
```

Comment:
I'm not sure I follow - it still seems like the inferred value isn't actually consumed anywhere?